<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shapes Demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>2</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <item row="5" column="0" colspan="3">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" rowspan="5">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2" rowspan="5">
       <widget class="QFrame" name="frameDraw">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>265</width>
          <height>287</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QFrame" name="frameDraw2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>237</width>
             <height>267</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>237</width>
             <height>267</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame#frameDraw2{border: 1px inset black;}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="DrawArea" name="areaDraw" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>235</width>
                <height>265</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>235</width>
                <height>265</height>
               </size>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::NoContextMenu</enum>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <widget class="ContentFilterSelector" name="widget_contentFilter" native="true">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="geometry">
                <rect>
                 <x>60</x>
                 <y>90</y>
                 <width>61</width>
                 <height>71</height>
                </rect>
               </property>
               <property name="contextMenuPolicy">
                <enum>Qt::NoContextMenu</enum>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="bt_subscribe">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Subscribe</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="bt_publish">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Publish</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_endpoints">
       <attribute name="title">
        <string>Endpoints</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTableView" name="tableEndpoint">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabOutput">
       <attribute name="title">
        <string>Output</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextBrowser" name="text_Output"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>379</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="actionParticipant"/>
   </widget>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>Control</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionUser_Manual"/>
    <addaction name="actionInteroperability_Troubleshooting"/>
   </widget>
   <addaction name="menuControl"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop (delete all)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDelete_Enpoint">
   <property name="text">
    <string>Delete Enpoint</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionUser_Manual">
   <property name="text">
    <string>User Manual</string>
   </property>
  </action>
  <action name="actionInteroperability_Troubleshooting">
   <property name="text">
    <string>Interoperability Troubleshooting</string>
   </property>
  </action>
  <action name="actionParticipant">
   <property name="text">
    <string>Participant Configuration</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DrawArea</class>
   <extends>QWidget</extends>
   <header>eprosimashapesdemo/qt/DrawArea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ContentFilterSelector</class>
   <extends>QWidget</extends>
   <header>eprosimashapesdemo/qt/ContentFilterSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <signal>keyPressEvent(QKeyEvent*event)</signal>
 </slots>
</ui>
